src/test/unit/scala/com/evvo/utils/UntypedSerializationPackageTest.scala

```diff
@@ -0,0 +1,52 @@
+package com.evvo.utils
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream, ObjectInputStream, ObjectOutputStream}
+
+import org.scalatest.{Matchers, WordSpec}
+
+class UntypedSerializationPackageTest extends WordSpec with Matchers {
```

Comment:
If we can delete the UntypedSerializationPackage class, we can remove these tests too